graphics/libjpeg/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        thorfdbg libjpeg c719010a26ce0c666e98b2acf924ad5fc24b4f5d
+# see https://github.com/thorfdbg/libjpeg/blob/master/README
+version             1.70
+revision            0
+categories          graphics
+license             GPL-3
+maintainers         nomaintainer
+
+description         Complete implementation of 10918-1 (JPEG) from jpeg.org
+
+long_description    A complete implementation of 10918-1 (JPEG) coming \
+                    from jpeg.org (the ISO group) with extensions for \
+                    HDR standardized as 18477 (JPEG XT).
+
+checksums           rmd160  245dc39bc712770499a86d904e8aa332a2118c1a \
+                    sha256  62e8c8b1898a7e9195c90b673e6db0090b474372b4e7f485e471f37a08852e18 \
+                    size    560772
+
+distname            ${name}-${version}
+
+use_autoreconf      yes
+
+build.target        doc final \
+                    SETTINGS=clang
```

`SETTINGS=clang` is not a target so it should not be in `build.target`:

```suggestion
build.target        doc final
build.args          SETTINGS=clang
```

But it looks like `SETTINGS=clang` causes a set of clang-specific flags to be used, and clang is not the default compiler on all systems. Some systems use gcc and there is also a settings file for gcc-specific flags. So you probably want set `SETTINGS` to either `clang` or `gcc` based on whether `configure.cc` matches `clang`.

---

No such variable as `build.args` (or `build.post_args`):

```sh
Failed to parse file graphics/jpeg-xt/Portfile: can't read "build.args": no such variable
```

This setting must appear at the end of the `make` command.